config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+10
 
 # Target score for pipeline.py:
-0
+10
```

Comment:
I did not see the POS part, otherwise this work cannot event pretend to be for the highest mark

Reply:
Do you mean pos_frequency_pipeline.py? It did commit it